plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintFormatTask.kt

```diff
@@ -15,7 +15,7 @@ import javax.inject.Inject
 open class KtlintFormatTask @Inject constructor(
     objectFactory: ObjectFactory,
     projectLayout: ProjectLayout
-) : KtlintCheckTask(objectFactory, projectLayout) {
+) : BaseKtlintCheckTask(objectFactory, projectLayout) {
```

Comment:
I think this is an API breaking change because `tasks.withType<KtlintCheckTask>` will now behave differently.